File: VideoWeb/VideoWeb.EventHub/Models/UpdateEndpointsDto.cs
using System;
using System.Collections.Generic;
using VideoWeb.Contract.Responses;

namespace VideoWeb.EventHub.Models
{
    public class UpdateEndpointsDto
    {
        public IList<VideoEndpointResponse> ExistingEndpoints { get; set; } = new List<VideoEndpointResponse>();
        public IList<VideoEndpointResponse> NewEndpoints { get; set; } = new List<VideoEndpointResponse>();
        public IList<Guid> RemovedEndpoints { get; set; } = new List<Guid>();
    }
}

File: VideoWeb/VideoWeb/ClientApp/src/app/shared/models/update-endpoints-dto.ts
import { Guid } from 'guid-typescript';
import { VideoEndpointResponse } from '../../services/clients/api-client';

/* tslint:disable */
/* eslint-disable */
export class UpdateEndpointsDto {
    existing_endpoints: VideoEndpointResponse[];
    new_endpoints: VideoEndpointResponse[];
    removed_endpoints: Guid[];
}
